### PR TITLE
Initial action code

### PR DESCRIPTION
**Pull request by ds-ms**

### CODE REVIEW

File: src/kubectl.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    const kubectlPath = path.join(cachedToolpath, kubectlToolName + getExecutableExtension());
+    fs.chmodSync(kubectlPath, '777');
+    return kubectlPath;
+}
+
+export async function kubectlEvalLint(manifests: string[]) {
+    let toolPath = await downloadKubectl(await getStableKubectlVersion());
+    for (let i = 0; i < manifests.length; i++) {
+        const manifest = manifests[i];
+        let toolRunner = new ToolRunner(toolPath, ['apply', '-f', manifest, '--server-dry-run']);
```
Comment (vithati) [inline src/kubectl.ts]: Add namespace.

File: src/kubectl.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        cachedToolpath = await toolCache.cacheFile(kubectlDownloadPath, kubectlToolName + getExecutableExtension(), kubectlToolName, version);
+    }
+
+    const kubectlPath = path.join(cachedToolpath, kubectlToolName + getExecutableExtension());
+    fs.chmodSync(kubectlPath, '777');
+    return kubectlPath;
+}
+
+export async function kubectlEvalLint(manifests: string[]) {
+    let toolPath = await downloadKubectl(await getStableKubectlVersion());
```
Comment (vithati) [inline src/kubectl.ts]: validate connection is set or not.  Also validate server version.

File: action.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+name: 'Lint k8s manifest files'
```
Comment (vithati) [inline action.yml]: Add Readme for action.

File: src/kubectl.ts [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+    const kubectlPath = path.join(cachedToolpath, kubectlToolName + getExecutableExtension());
+    fs.chmodSync(kubectlPath, '777');
+    return kubectlPath;
+}
+
+async function validateConnection(toolPath: string) {
+    let toolRunner = new ToolRunner(toolPath, ['version'], { ignoreReturnCode: true });
+    const code = await toolRunner.exec();
+    if (code) {
```
Comment (vithati) [inline src/kubectl.ts]: It is good to have server version check here or have note in readme.

Comment (ds-ms) [reply src/kubectl.ts]: I'll add it to README

**Pull request merged**